src-java/testing/performance-tests/src/test/groovy/org/openkilda/performancetests/spec/benchmark/ConcurrentFlowCreateSpec.groovy

```diff
@@ -0,0 +1,64 @@
+package org.openkilda.performancetests.spec.benchmark
+
+import static groovyx.gpars.GParsPool.withPool
+
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.messaging.payload.flow.FlowState
+import org.openkilda.model.cookie.Cookie
+import org.openkilda.northbound.dto.v2.flows.FlowRequestV2
+import org.openkilda.performancetests.BaseSpecification
+import org.openkilda.performancetests.helpers.TopologyBuilder
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch
+
+import spock.lang.Shared
+import spock.lang.Unroll
+
+class ConcurrentFlowCreateSpec extends BaseSpecification {
+    @Shared
+    def r = new Random()
+
+    @Unroll
+    def "Flow creation (concurrent) on mesh topology"() {
+        given: "A mesh topology"
+        def topo = new TopologyBuilder(flHelper.fls,
+                preset.islandCount, preset.regionsPerIsland, preset.switchesPerRegion).buildMeshes()
+        topoHelper.createTopology(topo)
+        flowHelperV2.setTopology(topo)
+
+        when: "A source switch"
```

Comment:
as for me would be better to move it to the `and` block
for example:
```
given: "A mesh topology"
and: "A source switch for the flows"
when: "Create flows"
```

Reply:
Make sense. Moved.